meta-xilinx-standalone/recipes-bsp/embeddedsw/embeddedsw/makefile-skip-copy_bsp.sh.patch

```diff
@@ -7,41 +7,41 @@ YP integration specific
 
 Signed-off-by: Mark Hatle <mark.hatle@xilinx.com>
 
-diff --git a/lib/sw_apps/zynqmp_pmufw/src/Makefile b/lib/sw_apps/zynqmp_pmufw/src/Makefile
-index 1750c0a329..17f6a545ea 100644
---- a/lib/sw_apps/zynqmp_pmufw/src/Makefile
-+++ b/lib/sw_apps/zynqmp_pmufw/src/Makefile
-@@ -28,8 +28,8 @@ $(EXEC): $(LIBS) $(OBJS) $(INCLUDES)
+diff --git a/lib/sw_apps/versal_plm/src/versal/Makefile b/lib/sw_apps/versal_plm/src/versal/Makefile
```

Comment:
IMHO this sould be a new patch instead of repairing the old one. v2022.1 still contains the old file path and this will probably break builds for v2022.1

Reply:
Thanks, changed to the selecting patch file according to ESW_VER which hopefully alleviates the situation for 2022.1 users.